### PR TITLE
{data}[intel/2018a] NCL v6.5.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/g2lib/g2lib-3.1.0-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    '8a2de259de82094c5867f8d7945359f211592a4a503f9ed65dc60469337414e7',  # g2lib-3.1.0.tar
+    '0674b3db862aed0885d5213288892791646749524604239100f90c0d1d181f74',  # g2lib-3.1.0_fix-makefile.patch
+]
+
+dependencies = [
+    ('JasPer', '1.900.1'),
+    ('libpng', '1.6.34'),
+]
+
+#buildopts = 'CFLAGS="$CFLAGS -DLINUXG95 -D__64BIT__" FFLAGS="$FFLAGS -fpp -I."'
```
Comment (boegel) [inline easybuild/easyconfigs/g/g2lib/g2lib-3.1.0-intel-2018a.eb]: note to self: clean this up...

File: easybuild/easyconfigs/h/HDF/HDF-4.2.13-GCCcore-6.4.0-libjpeg-turbo-1.5.3.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'HDF'
+version = '4.2.13'
```
Comment (boegel) [inline easybuild/easyconfigs/h/HDF/HDF-4.2.13-GCCcore-6.4.0-libjpeg-turbo-1.5.3.eb]: note to self: there's a newer version of `HDF`, see https://portal.hdfgroup.org/display/support/Download+HDF4

File: easybuild/easyconfigs/h/HDF/HDF-4.2.13-GCCcore-6.4.0-libjpeg-turbo-1.5.3.eb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+libjpeg_turbo_ver = '1.5.3'
+versionsuffix = '-libjpeg-turbo-%s' % libjpeg_turbo_ver
+
+homepage = 'http://www.hdfgroup.org/products/hdf4/'
```
Comment (boegel) [inline easybuild/easyconfigs/h/HDF/HDF-4.2.13-GCCcore-6.4.0-libjpeg-turbo-1.5.3.eb]: note to self: outdated

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
node3126.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/309513858a69d8350597e402c5ede6af for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b8997433ba14eb9f77387791d966deb8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2002.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/64af8e2d06ae6b98eacf5f424e3cdc7c for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 8 out of 11 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/89788aad10ed1ab4fd9cf5114b7c7198 for a full test report.

**Comment by boegel** [comment]
@verdurin `xproto-7.0.31-GCCcore-6.4.0.eb` failed because of a (temporary?) download problem...

Can you please try again? Re-downloading the sources for `xproto-7.0.31-GCCcore-6.4.0.eb` works fine for me now...

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/7de0d7159e56d9207359b922a439bd9c for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**